.github/workflows/ci.yaml

```diff
@@ -32,7 +34,7 @@ jobs:
           java-version: '11'
           cache: 'gradle'
       - name: Install Buf CLI
-        run: go install github.com/bufbuild/buf/cmd/buf@v1.50.0
+        run: go install github.com/bufbuild/buf/cmd/buf@v1.57.2
```

Comment:
Would be good to bump this to [1.58](https://github.com/bufbuild/buf/releases/tag/v1.58.0), which updates the protovalidate lint rule.

`make ci` passes with 1.58 for me - I just think that it's worth updating to ensure that we don't accidentally slip in an example that violates the new lint rule.